pyroma/tests.py

```diff
@@ -88,6 +88,8 @@ def _proxy_method(*args, **kw):
         return _proxy_method
 
     def __getattr__(self, attr):
+        if attr.startswith("_"):
+            raise AttributeError()
```

Comment:
I'd suggest documenting this with a comment, and adding an appropriate error message to the AttributeError stating the name of the attribute and explaining what went wrong, like the others all have.

Reply:
Also, should this error on all private or dunder attributes, all dunder attributes or only getattr calls to itself?

Reply:
> Also, should this error on all private or dunder attributes, all dunder attributes or only getattr calls to itself?

I'm not sure. I think the only thinks that are strictly necessary are the names like `_data` that are used inside the implementation of `__getattr__` itself. But I'm not actually familar with this code base, I just observed that this was sufficient to get the tests to pass.

Reply:
I pushed a new version addressing your comments.

Reply:
LGTM now, thanks.